docs/Release_notes.md

```diff
@@ -1,6 +1,199 @@
+-	PR 13: Added wave JSON configurations for SDSM.
+
+### **CARLA Sensor Library** 
+
+The carla-sensor-lib is a new repository and docker image for this release. This library provides functions to create CARLA sensors with noise modeling and retrieve detection information from created sensors. Additionally this repository creates a docker image which deploys and XML RPC server that allows clients to create sensor in CARLA and poll their detections. This new feature is provided to enable object detection for CDASim deployment using CARLA's existing sensor and object representation. The data will be feed to the CARMA-Platform via the CARLA CARMA Integration tool, and to CDASim to expose the data to other integrated simulators or software systems under test.
+
+### **CARLA ScenarioRunner** 
+
+The scenario-runner is a new repository and docker image for this release. This directory contains custom ScenarioRunner scenario configurations to facilitate integration testing. This still a work in progress, so the scenarios serve more as example references. ScenarioRunner's code resides in the srunner Python package, and the scenario_runner.py script is responsible for launching scenarios. 
+
+### **EVC-SUMO (New Private Repository)** 
+
+This EVC-sumo is a new private repository. Econolite Virtual Controller (EVC)-SUMO bridge tool that is a part of XiL co-simulation tool. This bridge acts as a mediator between EVC (via the PyEOS Python package) and SUMO, integrating the exchange of traffic light status and detector status.
+Currently, in order to build and use the docker image, the tool needs proprietary github-token accessible to the members of the usdot-fhwa-stol organization due to Econolite's license.
+
+### **Multiple Object Tracking** 
+
+Multiple Object Tracking is a new repository that holds a library to support cooperative perception. This library is intended for tracking multiple objects from multiple sources where the input is abstracted as a standard cooperative perception object tracking interface. For example, object level data can come from multiple sources including J2334 Sensor Data Sharing Message, Basic Safety Message, or local perception. This enables the library to be deployed in different road actors such as C-ADS equipped vehicles or the infrastructure with the help of message adapters that are suited for respective middleware. 
+The library exposes multiple submodules and functionalities adopted from the architectural and algorithmic advancements made by the sensor fusion community. Example usage of this library to fully execute multiple object tracking pipeline is implemented in CARMA Platform here.   
+
+### **CARMA Builds** 
+
+CARMA Builds is a new component of the CARMA ecosystem, which enables a coordination among different transportation users. This component provides Docker images used to build other projects within CARMA. 
```

Comment:
CARMA Builds is a new addition to the CARMA ecosystem, designed to facilitate Docker image build coordination among various transportation users. This component provides essential Docker images for building other projects within CARMA, streamlining the development process and enhancing collaboration across different transportation initiatives.